api/v1alpha1/airflowbase_types.go

```diff
@@ -293,6 +293,8 @@ type AirflowUISpec struct {
 	// Resources is the resource requests and limits for the pods.
 	// +optional
 	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
+
+	EnableRoutes bool `json:"enableroutes,omitempty"`
```

Comment:
Can you add comment about what is this for (like the other fields have)

Reply:
Added comments